samples/packaged/CopilotChat/SKWebApi/Program.cs

```diff
@@ -62,7 +71,7 @@ private static void AddSemanticKernelServices(this IServiceCollection services,
         kernelConfig.AddCompletionBackend(completionConfig);
         ISemanticTextMemory memory = NullMemory.Instance;
         AIServiceConfig embeddingConfig = configuration.GetSection("EmbeddingConfig").Get<AIServiceConfig>();
-        if (embeddingConfig.IsValid())
+        if (embeddingConfig?.IsValid() == true)
```

Comment:
Does IsValid() not already allow the true/false comparison?

Reply:
It does, but notice the "?".  This checks whether we are null, true, or false